internal/task/task/fs/mount.go

```diff
@@ -153,11 +182,28 @@ func NewMountFSTask(curvradm *cli.CurveAdm, cc *client.ClientConfig) (*task.Task
 		ExecWithSudo:      false,
 		ExecInLocal:       true,
 	})
+
+	t.AddStep(&step.SyncFile{ // sync tools config
+		ContainerSrcId:    &containerId,
+		ContainerSrcPath:  fmt.Sprintf("%s/conf/tools.conf", root),
+		ContainerDestId:   &containerId,
+		ContainerDestPath: LAYOUT_CURVEFS_TOOLS_CONFIG_SYSTEM_PATH,
+		KVFieldSplit:      CLIENT_CONFIG_DELIMITER,
+		Mutate:            newToolsMutate(cc, CLIENT_CONFIG_DELIMITER),
+		ExecWithSudo:      false,
+		ExecInLocal:       true,
+	})
 	t.AddStep(&step.StartContainer{
 		ContainerId:  &containerId,
 		ExecWithSudo: false,
 		ExecInLocal:  true,
 	})
+	t.AddStep(&step.ContainerExec{ // create fs
```

Comment:
should we execute `create fs` after mount done, is it works?

Reply:
> should we execute `create fs` after mount done and is it works?

In this case, any process that accesses the mounted directory will get stuck. If the creation fails, should not continue.